code/modules/research/destructive_analyzer.dm

```diff
@@ -65,5 +65,5 @@ Note: Must be placed within 3 tiles of the R&D Console
 		user << "<span class='notice'>You add the [O.name] to the [src.name]!</span>"
 		flick("d_analyzer_la", src)
 		spawn(10)
+			icon_state = "d_analyzer_l"

```

Comment:
Not having it caused a bug that showed DA to be empty when there was an item inside since the sprite wasn't changing to DA_loaded icon. This is just the way it used to be in the past, I think the attackby things removed this.